server/chart_config/climate.json

```diff
@@ -91,6 +110,53 @@
     },
     "isChoropleth": true
   },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-carbon_dioxide_emissions_nonbiogenic",
+    "title": "Carbon dioxide emissions (non-biogenic)",
+    "description": "Annual amount of carbon dioxide emissions (non-biogenic)",
+    "statsVars": ["Annual_Emissions_CarbonDioxide_NonBiogenic"],
+    "unit": "t",
+    "relatedChart": {
+      "scale": false
+    }
+  },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-methane_emissions_nonbiogenic",
+    "title": "Methane emissions (non-biogenic)",
+    "description": "Annual amount of methane emissions (non-biogenic)",
```

Comment:
For the emissions, should we add "(large facilities)" ?

Reply:
Added "large facilities" to all emission chart